client/src/pages/HomePage/HomePage.js

```diff
@@ -1,10 +1,49 @@
-import React from 'react';
+import React, { useContext, useState, useEffect } from 'react';
+import axios from 'axios';
+import { StateContext } from '../../App';
+import { Box, Typography } from '@mui/material';
+import AnimalTypeBox from './components/AnimalTypeBox';
+import {
+  homePageContainer,
+  animalTypeContainer,
+  titleText,
+} from '../../styles/home/animalTypeBoxStyles';
 
 const HomePage = () => {
+  const { state } = useContext(StateContext);
+  const [animalTypes, setAnimalTypes] = useState([]);
+
+  const config = {
```

Comment:
rename to `axiosConfig` because naming it to `config` is unclear (which config is it referring to)